README.md

```diff
@@ -347,37 +347,37 @@ interface Event {
 }
 ```
 
-The directives (`Root = app`, `View = view` and `Zone = zone`) are responsible for defining their respective attributes for the data structure. Whenever you dispatch an event, these values will be retrieved based on the component hierarchy, for example:
+The directives (`Root = app`, `View = view` and `Element = elementTree`) are responsible for defining their respective attributes for the data structure. Whenever you dispatch an event, these values will be retrieved based on the component hierarchy, for example:
 
 ```jsx
  <TrackingRoot name="my-app" onDispatch={console.log}>
    <TrackingView name="account">
     ...
-    <TrackingZone name="change-account-form">
+    <TrackingElement name="change-account-form">
      ...
-    </TrackingZone>
+    </TrackingElement>
    </TrackingView>
  <TrackingRoot>
 ```
 
-Would yield the following structure: `{ app: 'my-app', view: 'account', zone: 'change-account-form' }`.
+Would yield the following structure: `{ app: 'my-app', view: 'account', elementTree: ['change-account-form'] }`.
```

Comment:
I would add multiple nested TrackingElements here to illustrate how they are tracked in the `elementTree`.